cmd/thanos/rule.go

```diff
@@ -192,6 +212,8 @@ func runRule(
 	// back or the context get canceled.
 	queryFn := func(ctx context.Context, q string, t time.Time) (promql.Vector, error) {
 		var addrs []string
+		// Add addresses from static flag
+		addrs = append(addrs, queryAddrs...)
```

Comment:
Within `queryFn` do we want to remove duplicates if they exist.

Reply:
May even want to do this in #547 as we could supply a value in the file that its picked up by gossip as well if both systems are running.

Reply:
done in #547